<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InsertDialog</class>
 <widget class="QDialog" name="InsertDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>587</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Insert files</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/res.qrc"><normaloff>:app.ico</normaloff>:app.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="conflictsWidget" native="true">
     <layout class="QVBoxLayout" name="conflictsBox">
      <item>
       <widget class="QLabel" name="conflictsLabel">
        <property name="text">
         <string>The following files already exist:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="pboman3::ui::ConflictsList" name="conflictsList">
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="uniformRowHeights">
         <bool>true</bool>
        </property>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="compressWidget" native="true">
     <layout class="QVBoxLayout" name="compressBox">
      <item>
       <widget class="QLabel" name="compressLabel">
        <property name="text">
         <string>Choose files to compress:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="pboman3::ui::CompressList" name="compressList">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="uniformRowHeights">
         <bool>true</bool>
        </property>
        <property name="itemsExpandable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="pboman3::ui::InsertDialogButtons" name="buttons" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pboman3::ui::CompressList</class>
   <extends>QTreeView</extends>
   <header>ui/compresslist.h</header>
  </customwidget>
  <customwidget>
   <class>pboman3::ui::ConflictsList</class>
   <extends>QTreeWidget</extends>
   <header>ui/conflictslist.h</header>
  </customwidget>
  <customwidget>
   <class>pboman3::ui::InsertDialogButtons</class>
   <extends>QWidget</extends>
   <header>ui/insertdialogbuttons.h</header>
   <container>1</container>
   <slots>
    <signal>ok()</signal>
    <signal>cancel()</signal>
    <signal>back()</signal>
    <signal>next()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttons</sender>
   <signal>ok()</signal>
   <receiver>InsertDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>572</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttons</sender>
   <signal>back()</signal>
   <receiver>InsertDialog</receiver>
   <slot>back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>572</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttons</sender>
   <signal>cancel()</signal>
   <receiver>InsertDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>572</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttons</sender>
   <signal>next()</signal>
   <receiver>InsertDialog</receiver>
   <slot>next()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>572</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>next()</slot>
  <slot>back()</slot>
 </slots>
</ui>
